src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor(){
```

Comment:
Method names should be verbs or verb phrases to represent actions. Consider renaming `getRandomColor` to a more action-oriented name like `generateRandomColor`.